hrm/department_app/models/employees.py

```diff
@@ -0,0 +1,18 @@
+"""Django models"""
+from django.db import models
+from djmoney.models.fields import MoneyField
+
+
+class Employees(models.Model):
+    """Employees"""
+    Id = models.IntegerField('ID')
+    name = models.CharField('Name',max_length=50)
+    department = models.CharField('Department',max_length=50)
```

Comment:
The department field must be a Foreign Key.

Reply:
Fixed.